### ISSUE TITLE
Training problem

### ISSUE THREAD

**Issue by scott870430** [yun-liu/LIID#2]
Hi, I am interested in your project. Could you provide the code for training MIL framework (loss function and CAM) and the code for filtering proposals? 

Thanks.

**Comment by yuhuan-wu** [comment]
Thanks. I will send a e-mail to you very soon. Could I know your e-mail address?

**Comment by scott870430** [comment]
HI, thank you for your reply. Is your e-mail wuyuhuan (at) mail(dot)nankai(dot)edu(dot)cn)? I sent you an e-mail yesterday. My e-mail starts with scott. 

Thank you for your time.

**Comment by yuhuan-wu** [comment]
Code sent, thanks for your attention.

**Comment by yyyyojiro** [comment]
> HI, thank you for your reply. Is your e-mail wuyuhuan (at) mail(dot)nankai(dot)edu(dot)cn)? I sent you an e-mail yesterday. My e-mail starts with scott.
> 
> Thank you for your time.

Hi, could you send me a copy of the training code? Thank you.

**Comment by scott870430** [comment]
Hi, @yyyyojiro, what is your e-mail address?

**Comment by yyyyojiro** [comment]
***@***.***, thanks very much!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yun-liu/LIID"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月25日(星期六) 中午1:50
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yun-liu/LIID] Training problem (#2)





 
Hi, @yyyyojiro, what is your e-mail address?
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by scott870430** [comment]
Ah, I can't see your e-mail.

> ***@***.***, thanks very much!
> […](#)
> ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "yun-liu/LIID" ***@***.***&gt;; 发送时间:&nbsp;2021年9月25日(星期六) 中午1:50 ***@***.***&gt;; ***@***.******@***.***&gt;; 主题:&nbsp;Re: [yun-liu/LIID] Training problem (#2) Hi, @yyyyojiro, what is your e-mail address? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yyyyojiro** [comment]
emm, my e-mail address is shimmer.iiir(at)gmail(dot)com, thanks!




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yun-liu/LIID"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月25日(星期六) 中午1:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yun-liu/LIID] Training problem (#2)





 
Ah, I can't see your e-mail.
  
@., thanks very much!
 …
 ------------------&nbsp;原始邮件&nbsp;------------------ 发件人: "yun-liu/LIID" @.&gt;; 发送时间:&nbsp;2021年9月25日(星期六) 中午1:50 @.&gt;; @.@.***&gt;; 主题:&nbsp;Re: [yun-liu/LIID] Training problem (#2) Hi, @yyyyojiro, what is your e-mail address? — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.
  
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by scott870430** [comment]
@yyyyojiro, code sent!

**Comment by callmelaoyi** [comment]
Hi, I preciate your brilliant Idea. Could you provide the code for training MIL framework. Thank you very much.
My e-mail address is yuan21207921(at)126(dot)com

**Comment by scott870430** [comment]
@callmelaoyi, code sent!
